crates/marketplace/src/testing/mod.rs

```diff
@@ -96,3 +108,128 @@ async fn start_builder_state(
 
     (senders, global_state)
 }
+
+// get transactions submitted in previous rounds, [] for genesis
```

Comment:
I think we should be doing `rustdoc` style comments above `function`s, `method`s and `structs` / `enum`s for clarity.  That way they appear when we interactive with the symbol and an LSP.

Reply:
I made some updates, but just to confirm, the comment here is for defining the transactions variable, not for a function, method, or struct. So there’s no need to change this specific line right?